### DISCUSSION TITLE
Does WC have an API endpoint that I can hit to list all of the wallet options? (Discord Question)

### DISCUSSION THREAD

**Discussion by GrahamMcBain** [WalletConnect/walletconnect-monorepo#1553] [Developer Support] [answered]
**_Initially posted in discord by @FounderCasey_**

**Response from Graham:** "API I'm not sure (I'm only a week in I'll ask and get back to you) but for now you can check out the explorer https://explorer.walletconnect.com/"

**Response from Casey:** "Thanks, Graham! I basically want the API that feeds the explorer 🙂 

For reference, we'd like to grab the current list of supported wallets to ensure we're showing what's accurate on our end. Also, if it's not possible could I get a json output from the API, that way I don't have to scrape? "

**Comment by bkrem** [comment, marked as answer]
Yes, we have a corresponding public API: `explorer-api.walletconnect.com`

Documentation can be found here: https://docs.walletconnect.com/2.0/introduction/cloud-explorer#cloud-explorer-api

This is the same API that powers Web3Modal and our Explorer site https://explorer.walletconnect.com
